src/main/java/com/synopsys/integration/alert/web/config/JobConfigController.java

```diff
@@ -288,7 +287,15 @@ public JobConfigController(final JobConfigActions jobConfigActions, final Respon
     private boolean hasRequiredPermissions(Collection<FieldModel> fieldModels, BiFunction<String, String, Boolean> permissionChecker) {
         return fieldModels
                    .stream()
-                   .allMatch(model -> permissionChecker.apply(model.getContext(), model.getDescriptorName()));
+                   .allMatch(model -> permissionChecker.apply(model.getContext(), trimCustomMessageName(model.getDescriptorName())));
+    }
+
+    private String trimCustomMessageName(String descriptorName) {
+        final int index = descriptorName.indexOf(JobConfigActions.CUSTOM_MESSAGE);
```

Comment:
What is with this?

Reply:
Why are we trimming the descriptor name? When would he descriptor name ever include `-CUSTOM_MESSAGE` as a suffix?

Reply:
The UI is adding a third descriptor in the Job Configuration as payload.